### DISCUSSION TITLE
PCL & WebView

### DISCUSSION THREAD

**Discussion by Yue-cn** [Meloong-Git/PCL#2562] [讨论]
![Animation (32)](https://github.com/Hex-Dragon/PCL2/assets/79188080/766a415e-b420-442c-a490-e760e1360d00)

**Comment by Yue-cn** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/79188080/610f769e-093a-46e0-807e-0d50ac8b3517)
内嵌 WebView 也不是做不到，只是说很奇怪，因为是另创Hwnd，所以内容会溢出窗口

**Comment by MoYuan-CN** [comment]
额？

**Comment by so1ve** [comment]
要是有了内置的webview那皮肤预览就好做了

**Comment by mc-124** [comment]
这个应该不至于用dll吧？
